Getting-Started/Backoffice/Content-Templates/index.md

```diff
@@ -0,0 +1,66 @@
+# Content Templates
+Content Templates allow you to create a blueprint for new content nodes based on an existing node.
+
+## Create - Method 1
+
+Select the 'Content' menu: 
+
+![Content Menu](../images/01-Content-Menu.png)
```

Comment:
Going one level up with images won't work on the OUR documentation website as that according to @jmayntzhusen .  We need to have images in a subfolder of the current file.

Reply:
No problem, done!